src/stories/containers/TransparencyReport/components/TransparencyTransferRequest/useTransparencyTransferRequest.tsx

```diff
@@ -89,6 +98,44 @@ export const useTransparencyTransferRequest = (currentMonth: DateTime, budgetSta
     return result;
   }, [currentMonth, budgetStatements]);
 
+  const getTransferRequestTargetBalanceColumn = useCallback((wallet: BudgetStatementWalletDto) => {
+    const targetWithTimeSpan: Pick<BudgetStatementWalletTransferRequestDto, 'target' | 'walletBalanceTimeStamp'> =
+      {} as BudgetStatementWalletTransferRequestDto;
+    wallet.budgetStatementTransferRequest?.forEach((item: BudgetStatementWalletTransferRequestDto) => {
+      targetWithTimeSpan.target = item.target;
+      targetWithTimeSpan.walletBalanceTimeStamp = item.walletBalanceTimeStamp;
+    });
+
+    return targetWithTimeSpan;
+  }, []);
+
+  const getWalletBalanceTimeStamp = useCallback(() => {
+    if (!wallets || !wallets[0].budgetStatementTransferRequest) return '';
+    const timeStampAnyWallet = wallets[0]?.budgetStatementTransferRequest[0]?.walletBalanceTimeStamp;
+
+    if (!DateTime.fromISO(timeStampAnyWallet).isValid) {
+      return '';
+    }
+
+    const dateTime = DateTime.fromISO(timeStampAnyWallet);
+    const formatData = dateTime.toFormat('dd-LLL');
+
+    return formatData;
+  }, [wallets]);
+
+  const getTransferRequestTargetBalanceColumnTotal = useMemo(() => {
```

Comment:
Suggestion: this name looks like a function but it is actually a number, maybe you should remove the `get` prefix

Reply:
I got you